components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -715,7 +715,7 @@ protected void initFile(String id) throws FormatException, IOException {
       seriesCount *= mosaics;
     }
 
-    ms0.imageCount = getSizeZ() * (isRGB() ? 1 : getSizeC()) * getSizeT();
+    ms0.imageCount = getSizeZ() * (isRGB() ? getSizeC()/3 : getSizeC()) * getSizeT();
```

Comment:
Yeah getEffectiveSizeC works based on the Z, T and image count. So although it would usually return the value we are using here, it can only be used once the image count is set so the code here is correct.